### DISCUSSION TITLE
NextJs and React Aria:Server Error

### DISCUSSION THREAD

**Discussion by birchrust** [adobe/react-spectrum#4653] [Q&A] [answered]
I create the button component by example, but there is a Server Error.
I don't know where is the problem, can help me?

code link:https://stackblitz.com/~/github.com/birchrust/demo
![image](https://github.com/adobe/react-spectrum/assets/121175143/cbc03a11-522b-4658-abb1-29631115684e)
![image](https://github.com/adobe/react-spectrum/assets/121175143/33c975a7-e032-4700-89d4-0c8fb247eb74)

**Comment by snowystinger** [comment, marked as answer]
Have you checked out some other discussions around nextjs? did any of them help?
https://github.com/adobe/react-spectrum/discussions/4471
https://github.com/adobe/react-spectrum/issues/3707

**Reply by birchrust** [reply]
thanks
